app/default-config.js

```diff
@@ -0,0 +1,26 @@
+export const defaultConfig = {
+  commands: {
+    label: {
+      permission: "author-or-member",
+      allowed_labels: [],
```

Comment:
I feel like it would be better to consistent at use camelCasing.
```suggestion
      allowedLabels: [],
```